### DISCUSSION TITLE
Will pinocchio v3.0.0 be available on pypi/conda?

### DISCUSSION THREAD

**Discussion by matheecs** [stack-of-tasks/pinocchio#2265] [General]
As of now, the version available on pypi/conda is 2.7. Will v3.0 be available?

**Comment by ManifoldFR** [comment]
Hello,

Our engineers are indeed working on the pypi and conda releases for 3.0 !
We've had some unexpected build issues for some platforms, but they're getting fixed :)

**Reply by jcarpent** [reply]
This is now released on Conda at least. Stay tuned for the other package managers.

**Reply by joao-pm-santos96** [reply]
Hello! After https://github.com/stack-of-tasks/pinocchio/issues/2270#issuecomment-2222357349, is there something I can help with?

**Comment by joao-pm-santos96** [comment]
Hello all! Any update on this? Is there something I can do to help?
Thank you for your hard work on this amazing software!

**Reply by ManifoldFR** [reply]
We have version 3.2.0 on conda-forge, don't know about pypi though

**Reply by joao-pm-santos96** [reply]
Yeah I know, but since v2.7.0 is available on pypi, I thought that the team would keep supporting it. 
Is there any reason on why not? Couldn't we use [manylinux](https://github.com/pypa/manylinux) to provide such wheels?
Please considere me just a beginner in these matters, trying to learn something new... :)

**Reply by nim65s** [reply]
The reason is that it requires a lot of time to maintain a coherent binary distribution on PyPI.
Currently, the infrastructure is broken for macos, because github updated its runners from intel to arm, and those does not seem to build correct binaries.
Also, it needs to be synchronized with other releases, so I'll do this with an upgrade of boost, and after https://github.com/humanoid-path-planner/hpp-fcl/pull/596 and coal release.
